terracumber_config/tf_files/SUSEManager-Head-NUE.tf

```diff
@@ -185,7 +185,7 @@ module "cucumber_testsuite" {
       install_salt_bundle = true
     }
     build-host = {
-      image = "sles15sp2o"
+      image = "sles15sp4o"
```

Comment:
We don't have a SP4 image in the Docker registries yet, this prevents us from moving away from SP2. There's a card about it.

Reply:
Well, we can have a docker build host as SP4 with a Docker image in it that is SP2. That's calling for problems, but "should work" in theory.

Reply:
Is this still relevant @Bischoff? I mean does it prevent us from merging this PR?

Reply:
Not sure. https://github.com/SUSE/spacewalk/issues/17619 is closed, not sure this was the card I was referring to.

I would say, let's just try and see what happens.

Reply:
I saw this PR from Oscar: https://github.com/uyuni-project/uyuni/pull/5393